bundle/Dockerfile

```diff
@@ -114,8 +113,8 @@ ENV DEVICE_PROVIDER_TYPE=plgd
 ENV DEVICE_OAUTH_SCOPES=profile,openid,offline_access
 ENV SDK_OAUTH_RESPONSE_MODE=form_post
 ENV SDK_OAUTH_SCOPES=openid
+ENV OWNER_CLAIM="sub"
 ENV SERVICE_CLIENT_CONFIGURATION_CLOUDAUTHORIZATIONPROVIDER="plgd"
-ENV SERVICE_CLIENT_CONFIGURATION_JWTCLAIMOWNERID="sub"
```

Comment:
`SERVICE_CLIENT_CONFIGURATION_JWTCLAIMOWNERID` doesn't have to match `OWNER_CLAIM`. You have a use-case when service uses sub - user id for a user filter, but authorization service - custom implementation like Identity Server uses different claim, e.g. Tenant.